next-cloudinary/src/components/CldVideoPlayer/CldVideoPlayer.tsx

```diff
@@ -135,8 +136,30 @@ const CldVideoPlayer = (props: CldVideoPlayerProps) => {
         }
       }
 
+      /* 
+      Parse the value passed to 'autoplay' or 'autoPlay';
+      if its a boolean or a boolean passed as string ("true") set it directly to browser standard prop autoplay else fallback to default; 
+      if its a string and not a boolean passed as string ("true") set it to cloudinary video player autoplayMode prop else fallback to default;
+      */
+      const autoplayFallback = autoplay || autoPlay ;
+      let autoPlayValue: boolean | 'true' | 'false' = false;
+      let autoplayModeValue: string = 'never';
+
+      if ( autoPlay && process.env.NODE_ENV === 'development' ) {
+        console.warn('Prop autoPlay will be removed in future versions, please use autoplay (lowercase "p")')
+      }
+
+      if (typeof autoplayFallback === 'boolean' || autoplayFallback === 'true' || autoplayFallback === 'false') {
+        autoPlayValue = autoplayFallback
+      }
+
+      if (typeof autoplayFallback === 'string' && autoplayFallback !== 'true' && autoplayFallback !== 'false') {
+        autoplayModeValue = autoplayFallback;
+      }
+
       let playerOptions: CloudinaryVideoPlayerOptions = {
-        autoplayMode: autoPlay,
+        autoplayMode: autoplayModeValue,
+        autoPlay: autoPlayValue,
```

Comment:
i think this needs to be a lowercase "p" as well https://cloudinary.com/documentation/video_player_api_reference#player_behavior